src/components/Table/Table.tsx

```diff
@@ -1,15 +1,23 @@
 import classNames from "classnames";
 import PropTypes from "prop-types";
-import React from "react";
+import React, { HTMLProps, ReactNode } from "react";
+
+type Props = {
+  children?: ReactNode;
+  className?: string;
+  expanding?: boolean;
+  responsive?: boolean;
+  sortable?: boolean;
+} & HTMLProps<HTMLTableElement>;
 
 const Table = ({
   children,
   className,
-  expanding,
-  responsive,
-  sortable,
+  expanding = false,
+  responsive = false,
+  sortable = false,
```

Comment:
With these last three being listed as optional we should probably also document their defaults as the other two do not have a default value.

Reply:
@hatched thanks for the review! when you say "document their default", can you explain what you had in mind? We don't use jsdoc in this project, and these components don't actually have storybooks (although arguably should).

Reply:
Yeah I didn't mean jsdoc specifically, I was more thinking about typedoc. But yeah if you could add a docblock to that effect that would be great.

Reply:
@hatched I'm going to hold off on adding jsdoc here, as there are couple of different approaches I think we should discuss in dev catchup first:

1. add a js/tsdoc block to the component
2. add jsdoc blocks directly to the Props type, setting defaults with `@default`, and generating proptypes from these (material-ui does this)